src/braket/aws/aws_session.py

```diff
@@ -177,7 +186,18 @@ def create_quantum_task(self, **boto3_kwargs) -> str:
         job_token = os.getenv("AMZN_BRAKET_JOB_TOKEN")
         if job_token:
             boto3_kwargs.update({"jobToken": job_token})
+        self.braket_client._client_config.user_agent_extra = (
+            f"({len(active_trackers())} Active Cost Trackers)"
+        )
```

Comment:
I don't think this would work. boto only looks at `user_agent_extra` when initializing the client, from what I can remember.

Reply:
You are correct. I've added a handler to update the user agent with updated count on Braket api calls instead